packages/visual-editor/src/components/Dropzone/Dropzone.tsx

```diff
@@ -71,6 +73,21 @@ export function Dropzone({
     },
     [resolveDesignValue],
   );
+  // To avoid a circular dependency, we create the recursive rendering function here and trickle it down
```

Comment:
```suggestion
```
it's not a recursive rendering in this case, so we can drop the copied message in here.